tree-construction/tests20.dat

```diff
@@ -514,3 +514,57 @@
 |       <math annotation-xml>
 |         encoding=" text/html "
 |     <div>
+
+#data
+<math><annotation-xml> </annotation-xml>
+#errors
```

Comment:
I found the errors that existed were very helpful when writing a parser, but I understand it's a pain to add and maintain them.